src/app/Components/email-preview/email-preview.component.ts

```diff
@@ -55,4 +57,10 @@ export class EmailPreviewComponent implements OnInit {
       });
     }
   }
+  openEmailDetail(data: any) {
```

Comment:
`(dataId: number | string)`

Reply:
I replaced (data: any) with (id: string), can't use (id: number | string) because this.active only accepts string type, otherwise must add type to active declaration.